Data/market-hours/market-hours-database.json

```diff
@@ -601,9 +601,330 @@
         }
       ],
       "saturday": [],
-      "holidays": []
+      "holidays": [
+        "01/01/2013",
+        "01/20/2013",
+        "01/21/2013",
+        "02/17/2013",
+        "02/18/2013",
+        "03/29/2013",
+        "05/26/2013",
+        "07/04/2013",
+        "09/01/2013",
+        "11/28/2013",
+        "12/25/2013",
+        "01/01/2014",
+        "01/19/2014",
+        "02/16/2014",
+        "04/18/2014",
+        "05/25/2014",
+        "07/04/2014",
+        "07/06/2014",
+        "08/31/2014",
+        "11/27/2014",
+        "12/25/2014",
+        "01/01/2015",
+        "01/18/2015",
+        "02/15/2015",
+        "04/03/2015",
+        "07/03/2015",
+        "09/06/2015",
+        "11/26/2015",
+        "12/25/2015",
+        "01/01/2016",
+        "01/17/2016",
+        "02/14/2016",
+        "03/25/2016",
+        "05/29/2016",
+        "07/03/2016",
+        "07/04/2016",
+        "09/04/2016",
+        "11/24/2016",
+        "01/01/2017",
+        "01/16/2017",
+        "02/19/2017",
+        "04/14/2017",
+        "05/28/2017",
+        "07/04/2017",
+        "09/03/2017",
+        "11/23/2017",
+        "12/25/2017",
+        "01/01/2018",
+        "01/14/2018",
+        "02/18/2018",
+        "03/30/2018",
+        "05/27/2018",
+        "07/04/2018",
+        "09/02/2018",
+        "11/22/2018",
+        "12/25/2018",
+        "01/01/2019",
+        "01/20/2019",
+        "02/17/2019",
+        "04/19/2019",
+        "05/26/2019",
+        "07/04/2019",
+        "09/01/2019",
+        "11/28/2019",
+        "12/25/2019",
+        "01/01/2020",
+        "01/19/2020",
+        "02/16/2020",
+        "04/10/2020",
+        "05/24/2020",
+        "07/03/2020",
+        "09/06/2020",
+        "11/26/2020",
+        "12/25/2020",
+        "01/01/2021"
+      ],
+      "earlyCloses": {
+        "03/28/2013": "14:00:00",
+        "07/03/2013": "12:00:00",
+        "11/27/2013": "13:20:00",
```

Comment:
I'm seeing this as `13:15:00` on my end; maybe we're using different data sources? I've attached the spreadsheet I'm sourcing data from in this comment:

https://www.dropbox.com/s/ujzj77vc99qdiqt/2013-thanksgiving.xls?dl=0

Reply:
Yep, using the same spreadsheet, just that some of the data was hard to read with the overarching column headings originally, but it's much clearer now I've gone through the following year's spreadsheets, which are much clearer!